cms/templates/cms/home_page.html

```diff
@@ -68,45 +74,150 @@
+                                  More about {{ program }}
+                                </a>
+                              </p>
+                            {% endif %}
+                          </div>      
+                        </div>
+                        {% if programpage and programpage.thumbnail_image %}
+                          {% image programpage.thumbnail_image fill-690x530 as thumbnail_image %}
+                          <img src="{{ thumbnail_image.url }}" alt=""
+                               >
+                        {% else %}
+                          <img src="{% static 'images/course-thumbnail.png' %}"
+                               alt="" class="program-default-image">
+                        {% endif %}
+                      {% if programpage %}
+                        </a>
+                      {% endif %}
+                    </div>
+                  </li>
+                {% endfor %}
+        
+              </ul>
               </div>
             </div>
           </section>
         </div>
       </div>
 
+      <section class="row reasons-section">
+        <div class="centered-content">
+          <h3>Why should I enroll?</h3>
+          <div class="enroll-reason col-md-3">
+            <div class="reasons-images-wrap">
+               <img src="{% static 'images/online.png' %}" alt="">
+            </div>
+
+            <h4>Convenient &amp; Flexible</h4>
+            <p>Take online courses on a schedule that fits your needs, and meet motivated learners who share your interests.</p>
+          </div>
+          <div class="enroll-reason col-md-3">
+            <div class="reasons-images-wrap">
+              <img src="{% static 'images/blended.png' %}" alt="">
+            </div>
+
+            <h4>Accelerate Your Master’s</h4>
+            <p>A MicroMasters program is a faster path to a master’s degree from MIT and other universities, and is recognized by employers for providing deep learning in specific career fields.</p>
+
+          </div>
+          <div class="enroll-reason col-md-3">
+            <div class="reasons-images-wrap">
+              <img src="{% static 'images/cost.png' %}" alt="">
+            </div>
+             <h4>Save Money &amp; Time</h4>
+            <p>A MicroMasters program is an affordable option for professionals who can’t take off time to earn a traditional master’s degree.</p>
+          </div>
+          <div class="enroll-reason col-md-3">
+            <div class="reasons-images-wrap">
+              <img src="{% static 'images/globe-icon.png' %}" alt="">
+            </div>
+             <h4>Grow Your Network</h4>
+            <p>After you earn your MicroMasters® program credential, you will become an affiliate of the MIT Alumni Association.{% if benefits_url %} <a href="{{ benefits_url }}">Learn More</a>{% endif %}</p>
+          </div>
+        </div>
+        <div class="benefits">
+          <a href="#">Learn more about these benefits</a>
```

Comment:
This should link to the benefits page, which will be created in the CMS, i.e. /benefits/